grakn-kb/src/main/java/ai/grakn/factory/GraknSessionImpl.java

```diff
@@ -93,10 +95,7 @@
             ThreadFactory namedThreadFactory = new ThreadFactoryBuilder()
                     .setNameFormat("commit-log-subbmit-%d").build();
             commitLogSubmitter = Executors.newSingleThreadScheduledExecutor(namedThreadFactory);
-            commitLogSubmitter.scheduleAtFixedRate(() -> {
-                submitLogs(tx);
-                submitLogs(txBatch);
-            }, 0, LOG_SUBMISSION_PERIOD, TimeUnit.SECONDS);
+            commitLogSubmitter.scheduleAtFixedRate(this::submitLogs, 0, LOG_SUBMISSION_PERIOD, TimeUnit.SECONDS);
```

Comment:
When/how do we submit the commit logs for batch transactions?

Reply:
Both the normal logs and batch logs are put in the same `CommitLogHandler` so we only need one submission